src/cli/repo/repo.go

```diff
@@ -63,7 +75,7 @@ func repoCmd() *cobra.Command {
 	return &cobra.Command{
 		Use:   "repo",
 		Short: "Manage your repositories",
-		Long:  `Initialize, configure, and connect to your account backup repositories.`,
+		Long:  `Initialize, configure, connect and update to your account backup repositories.`,
```

Comment:
```suggestion
		Long:  `Initialize, configure, connect and update to your account backup repositories`,
```

Reply:
Full stop was present earlier. Do we want to remove it? I can see it is present at some places and missing at others.